recipe/meta.yaml

```diff
@@ -6,28 +6,32 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: fb4f1d47ee60efe87d4f6f0ebb5f70b9760db2b2574c59c8e8912be4ebd464c9
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ python_min }}
     - pip
     - setuptools >=61.0
   run:
-    - python >=3.8
+    - python >={{ python_min }}
     - setuptools >=61.0
+    # remove after python_min hits 3.11
+    # keeping this noarch: python
+    - tomli
```

Comment:
Was the previous build patched or removed? Other packages cannot run pip-check otherwise. The solver may prefer the build number with less dependencies sometimes.